nautobot-app-ssot/{{ cookiecutter.project_slug }}/{{ cookiecutter.app_name }}/tests/test_{{ cookiecutter.system_of_record_slug }}_adapter.py

```diff
@@ -32,7 +32,7 @@ def setUp(self):  # pylint: disable=invalid-name
 
         self.job = {{ cookiecutter.system_of_record_camel }}DataSource()
         self.job.job_result = JobResult.objects.create(
-            name=self.job.class_path, obj_type=ContentType.objects.get_for_model(Job), user=None, job_id=uuid.uuid4()
+            name=self.job.class_path, user=None, id=uuid.uuid4()
```

Comment:
Should be able to leave `id` out entirely and just let it be auto-assigned. Similarly, `None` should be the default for the `user` foreign key:

```suggestion
            name=self.job.class_path
```

Reply:
I'll test to verify and push new changes it it works